backend/app/src/pong/game.gateway.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  WebSocketGateway,
+  SubscribeMessage,
+  MessageBody,
+  WebSocketServer,
+  ConnectedSocket,
+} from '@nestjs/websockets';
+import { Server, Socket } from 'socket.io';
+import { PositionDto } from './dto/position.dto';
+import { GameService } from './game.service';
+import { SchedulerRegistry } from '@nestjs/schedule';
+import { JwtAuthGuard } from 'src/auth/guard/jwt.auth-guard';
+import { UseGuards } from '@nestjs/common';
+import { Game, Status } from './entities/position.entity';
+import { GetCurrentUserId } from 'src/common/decorators/getCurrentUserId.decorator';
+
+@WebSocketGateway({
+  cors: {
+    origin: [
+      'http://localhost',
```

Comment:
Do we need all these origins ? 
origin: "http://localhost:8080" should be enough no ?

Reply:
or just "http://localhost" ?

Reply:
This. is. resolved.